iot-e2e-tests/common/src/main/java/com/microsoft/azure/sdk/iot/common/helpers/IotHubServicesCommon.java

```diff
@@ -85,6 +98,36 @@ public static void sendMessages(InternalClient client,
         }
     }
 
+    public static void sendSecurityMessages(InternalClient client,
+                                            IotHubClientProtocol protocol,
+                                            final long RETRY_MILLISECONDS,
+                                            final long SEND_TIMEOUT_MILLISECONDS,
+                                            List<Pair<IotHubConnectionStatus, Throwable>> statusUpdates) throws IOException, InterruptedException
+    {
+        try
+        {
+            client.open();
+            if (statusUpdates != null)
+            {
+                confirmOpenStabilized(statusUpdates, 120000, client);
+            }
+
+            // Send the initial message
+            MessageAndResult messageToSend = new MessageAndResult(new Message("test message"), IotHubStatusCode.OK_EMPTY);
+            sendMessageAndWaitForResponse(client, messageToSend, RETRY_MILLISECONDS, SEND_TIMEOUT_MILLISECONDS, protocol);
```

Comment:
Is it necessary to send a normal message first?

Reply:
Yes, I want to make sure the system is working properly first, then make sure that the message gets put in the ASC system.

---
In reply to: [322453857](https://github.com/Azure/azure-iot-sdk-java/pull/576#discussion_r322453857) [](ancestors = 322453857)